circle.yml

```diff
@@ -107,6 +109,7 @@ test:
       net/rpc/jsonrpc
       net/textproto
       net/url
+      os/user
```

Comment:
Would this list be better maintained as an exclusion list? Not that there are many new packages added to core... but still.

Reply:
I agree, that'd be better. Not as part of this PR, but let's make a follow up enhancement PR after this one is merged.